docs/build/index.md

```diff
@@ -0,0 +1,8 @@
+<a id='Main.Lapidary-1'></a>
+
+<a href='#Main.Lapidary-1'> # </a>**Constant**
```

Comment:
I forgot that this fallsback the the README. Which I think is going to be a bit circular here (if we generate the README!)...